src/sagemaker/jumpstart/artifacts/model_uris.py

```diff
@@ -50,6 +51,8 @@ def _retrieve_model_uri(
             artifact S3 URI.
         model_scope (str): The model type, i.e. what it is used for.
             Valid values: "training" and "inference".
+        instance_type (str): An instance type to optionally supply in order to get
+            model artifacts specific for the instance type. (Default: None).
```

Comment:
`Optional. The ML compute instance type for the specified scope. (Default: None).`